common/src/stack/command/stack/commands/add/pallet/__init__.py

```diff
@@ -198,6 +216,29 @@ def update_db(self, pallet_info, URL):
 			# Re-added the pallet. Update the URL.
 			self.db.execute('UPDATE rolls SET url=%s WHERE id=%s', (URL, rows[0][0]))
 
+	def perform_checksums(self, iso_name):
+		'''
+		Given an iso_name perform all the checksums given in the checksum argument (if needed)
+		'''
+		for checksum in self.checksum.split(','):
+			(checksum_type, checksum_value) = checksum.split(':')
+			if checksum_type in valid_checksums:
+				print(f'Checking {checksum_type} == {checksum_value}', end='... ', flush=True)
+				checksum_input = f'{checksum_value} {iso_name}\n'
+				proc = self._exec(
+					cmd=f'{checksum_type}sum -c',
+					shlexsplit=True,
+					input=checksum_input
+				)
+				if proc.returncode != 0:
+					print('Error!')
```

Comment:
There's no need to print successes/failures, the checking message above is enough, then silently proceed or fail loudly with CommandError.